apm-agent-plugins/apm-hibernate-search-plugin/apm-hibernate-search-plugin-common/src/main/java/co/elastic/apm/agent/hibernate/search/HibernateSearchHelper.java

```diff
@@ -0,0 +1,49 @@
+/*-
+ * #%L
+ * Elastic APM Java agent
+ * %%
+ * Copyright (C) 2018 - 2019 Elastic and contributors
+ * %%
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ * #L%
+ */
+package co.elastic.apm.agent.hibernate.search;
+
+import co.elastic.apm.agent.impl.transaction.Span;
+import co.elastic.apm.agent.impl.transaction.TraceContextHolder;
+
+public final class HibernateSearchHelper {
+
+    private HibernateSearchHelper() {
+
+    }
+
+    public static Span createAndActivateSpan(final TraceContextHolder traceContextHolder, final String query) {
+        Span span = traceContextHolder.createSpan().activate();
+
+        span.withType("db")
+            .withSubtype(HibernateSearchConstants.HIBERNATE_SEARCH_ORM_TYPE)
+            .withAction("request");
```

Comment:
Why `request`? 
I see a few better options: 
1. Use your current subtype and the instrumented method name as the action.
2. Use `hibernate` as subtype and `search` as action.
3. If you can tell the underlying search engine (eg Lucene or Elasticsearch) - we can consider this info as well...
4. Use `search` as type and `hibernate` or underlying data store as subtype, method  name as action.

For now, let's go with option 2, ie `db.hibernate.search` and see what my colleague thinks about this. @felixbarny  WDYT?

Reply:
For now added span information from option 2.

I think option 3 would be possible with version 6. I know which provider my applicatoin is using so this would be no information I would need. But could be different in other scenarios.